frontend/src/metabase/components/Icon.tsx

```diff
@@ -23,7 +23,7 @@ export const IconWrapper = styled.div<IconWrapperProps>`
   justify-content: center;
   width: 40px;
   height: 40px;
-  border-radius: 99px;
+  border-radius: 6px;
```

Comment:
I wonder if the change might cause visual regressions? Don't we expect some icons to be round?

Reply:
@mazameli Did I go overboard? I interpreted that from https://www.notion.so/metabase/Inconsistent-hover-effects-on-action-buttons-5e619a2f52e04765872e82f424260bdd we're gonna use rounded rectangles instead of circular buttons/icons.

Reply:
I'll take a look.